pkg/adapter/github/adapter.go

```diff
@@ -50,7 +50,12 @@ func New(sinkURI, ownerRepo string) (*Adapter, error) {
 	if err != nil {
 		return nil, err
 	}
-	a.ownerRepo = ownerRepo
+	a.source = sourcesv1alpha1.GitHubEventSource(ownerRepo)
```

Comment:
`a.source` is set regardless of whether `source` is parsable?

Reply:
yeah, for printing the error. It will fail afterwards anyway. I'll move it down.
Thanks!

Reply:
@rootfs saw your comment after the lgtm, so this change didn't make it in in. Let me create a quick one with that change